packages/vite/rollup.config.js

```diff
@@ -273,7 +273,7 @@ function licensePlugin() {
       )
       const licenses = new Set()
       const dependencyLicenseTexts = dependencies
-        .sort(({ name: nameA }, { name: nameB }) => (nameA > nameB ? 1 : -1))
+        .sort(({ name: nameA }, { name: nameB }) => (nameA > nameB) ? 1 : ((nameB>nameA) ? -1 : 0))
```

Comment:
**suggestion:** Maybe we can shortcut this to:

```suggestion
        .sort(({ name: nameA }, { name: nameB }) => nameA - nameB)
```

Reply:
We can only apply the shortcut for numbers, not for strings.

Reply:
This works for strings also :)
TS don't like that, but it's working in JS and we could disable the lint/ts warning for that

Reply:
```suggestion
        .sort(({ name: nameA }, { name: nameB }) => (nameA > nameB) ? 1 : ((nameB > nameA) ? -1 : 0))
```